src/components/LabeledText/AddressLink.tsx

```diff
@@ -10,45 +10,43 @@ import {
   LabeledLinkPropTypes
 } from './LabeledLink'
 
+/**
+ * AddressLink is a LabeledLink for actor addresses.
+ *
+ * When fetchAddress is false (default), both id and address can be
```

Comment:
2 Small nits:

1. Think the comment line here should be changed from: "but only the (robust) address will be shown if present" to "but the robust address takes precedent over the id address if they both exist"
2. i know this wasn't introduced in this PR, and it's probably inconsistent in other places too, but I feel like we should move towards naming consistency in:

```
address: {id: string, robust: string}
robust: string
id: string
```

The AddressLink uses `address` as a string, which can either be a robust or an ID, which feels inconsistent. 

Maybe we can add an optional `robust` param, and make the `address` type _either_ the GQL address type or a string, and handle both, so we can start using it correctly.

Happy to open a separate ticket for this thought if you agree